app/panels.php

```diff
@@ -42,8 +42,8 @@
         'width' => '100%',
       ),
       'choices' => [
-        'shortlist_web_team' => 'Shortlist Web Team',
-        'stylist_web_team' => 'Stylist Web Team',
+        'the-shortlist-web-team' => 'The Shortlist Web Team',
```

Comment:
Let's use the 'user' taxononmy and the ACF Taxonomy Field here instead of hardcoding:

Ref: https://www.advancedcustomfields.com/resources/taxonomy/